### PR TITLE
feat(amazonq): update streaming client

### PR DESCRIPTION
**Pull request by AnirudhKonduru**
Instructions for building and updating the client: https://github.com/aws/aws-toolkit-vscode/blob/master/docs/build.md

## Problem
The streaming client must be updated periodically (until this process is automated, or the client is published to NPM).

## Solution
Updated the client.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by damntrecky — APPROVED** [review]

File: src.gen/@amzn/codewhisperer-streaming/dist-cjs/CodeWhispererStreaming.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"use strict";
```
Comment (justinmk3) [inline src.gen/@amzn/codewhisperer-streaming/dist-cjs/CodeWhispererStreaming.js:1]: why is `/dist-cjs/` being committed? surely we don't need both the `.js` files and the `.ts` files?

File: src.gen/@amzn/codewhisperer-streaming/dist-es/runtimeConfig.shared.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { defaultRegionInfoProvider } from "./endpoints";
```
Comment (justinmk3) [inline src.gen/@amzn/codewhisperer-streaming/dist-es/runtimeConfig.shared.js:1]: same question about `/dist-es/`. if we are committing dist folders, why do we have the typescript project?

if we can avoid the typescript project, that's great. but if we can't, then there is no reason to commit its build results?

File: src.gen/@amzn/codewhisperer-streaming/tsconfig.es.tsbuildinfo [hunk 3]
```diff
@@ -0,0 +1 @@
...564,565,566,567,568,569,570,571,572,593,578,584,582,581,583,592,587,589,590,591,585,586,588,580,579,574,573,576,577,575,594,534,595,596,597,599,600,60,62,63,64,65,66,67,68,69,70,71,72,73,74,61,102,75,76,77,103,78,79,80,81,82,83,84,85,86,87,88,89,90,91,92,93,94,95,96,97,98,99,100,101,601,602,603,526,525,505,604,514,605,630,631,606,609,628,629,619,618,616,611,624,622,626,610,623,627,612,613,625,607,614,615,617,621,632,620,608,645,644,639,641,640,633,634,636,638,642,643,635,637,646,535,648,647,649,650,651,652,556,654,653,655,656,657,658,660,659,519,598,545,554,543,544,555,550,551,549,553,547,546,552,548,521,520,518,522,390,523,369,368,193,194,195,371,370,244,373,190,372,191,389,379,386,387,388,245,246,436,435,437,247,248,251,249,250,252,254,253,428,432,429,431,430,196,197,198,199,200,201,202,210,211,212,213,214,215,216,217,204,205,203,208,206,207,233,218,219,220,221,222,223,224,225,226,227,228,229,230,231,209,232,487,488,404,237,238,239,240,236,234,235,243,241,242,192,400,8,9,11,10,2,12,13,14,15,16,17,18,19,3,4,23,20,21,22,24,25,26,5,27,28,29,30,6,34,31,32,33,35,7,36,41,42,37,38,39,40,1,43,403,441,440,402,398,399,442,413,433,444,391,443,392,401,499,503,423,427,434]},"version":"4.9.5"}
```
Comment (justinmk3) [inline src.gen/@amzn/codewhisperer-streaming/tsconfig.es.tsbuildinfo:1]: why are the buildinfo files being committed

File: src.gen/@amzn/codewhisperer-streaming/dist-es/runtimeConfig.shared.js (hunk 2 above)
Comment (AnirudhKonduru) [reply src.gen/@amzn/codewhisperer-streaming/dist-es/runtimeConfig.shared.js]: This was an oversight, fixed now.

**Pull request merged**